### PR TITLE
[ECO-732] Validators Tab

### PR DESCRIPTION
**Pull request by zie1ony**
### Overview

Add Validators tab.

### Which JIRA ticket does this PR relate to?

https://casperlabs.atlassian.net/browse/ECO-732

### CODE REVIEW

File: packages/ui/src/components/Validators.tsx [hunk 1]
```diff
@@ -1,61 +1,187 @@
...
+      // Prepare era rows.
+      let rows = Object.keys(data.era_validators[eraId])
+        .map(validatorId => {
+          return {
+            validatorId: validatorId,
+            stakeStr: data?.era_validators[eraId][validatorId],
+            stakeNum: BigNumber.from(data?.era_validators[eraId][validatorId])
+          };
+        })
+        .sort((a, b) => compareBigNumbers(a.stakeNum, b.stakeNum));
```
Comment (AbnerZheng) [inline packages/ui/src/components/Validators.tsx:128]: Maybe you could use `.sort((a, b) => a.stakeNum.lt(b.stakeNum))` directly?

**Review by AbnerZheng — APPROVED** [review]
LGTM, nice cleanup.

File: packages/ui/src/components/Validators.tsx (hunk 1 above)
Comment (zie1ony) [reply packages/ui/src/components/Validators.tsx:128]: `.sort` requires -1, 0 or 1 to be returned. `lt` returns boolean.

**Pull request merged**